networkit/cpp/auxiliary/test/AuxGTest.cpp

```diff
@@ -37,7 +37,7 @@ class AuxGTest: public testing::Test{};
 
 TEST_F(AuxGTest, produceRandomIntegers) {
 	Aux::Random::setSeed(1, false);
-#if (LOG_LEVEL == LOG_LEVEL_TRACE)
+#ifndef NETWORKIT_RELEASE_LOGGING
```

Comment:
AFAICS the main use of the `#ifdef` is to prevent unused variable warnings when `TRACE` or `DEBUG` is compiled out. A if with constant argument will not help in this case as the dead code elimination will only be done in the compiler backend, after the unused variable warning has already been issued.

That being said, a lot of the code using the `#if LOGLEVEL == [...]` idiom is quite questionable. I decided to only do syntactical changes to `LOG_LEVEL` in this PR but we can definitely take another look at those cases after the PR is merged.